src/components/FeatureViews/Accounts/TabComponents/Schedule.js

```diff
@@ -8,10 +10,37 @@ class Schedule extends Component {
         };
     }
 
+    // get instructors schdeule and parses it into one array of objcts 
+    getInstructorSchedule = () => {
+        let schedule = this.props.work_hours
+        let key = Object.keys(this.props.work_hours)
+
+        let instructorsSchedule = key.map((iKey) => {
+            return schedule[iKey]
+        })
```

Comment:
Here, you only want the *values* of the instructor's work hours - you don't care about the keys. Use [`Object.values()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/values) for this:

    let instructorSchedule = Object.values(this.props.work_hours);

Reply:
I just passed it into the FullCalendar events prop and changed the class component into a functional component. No need to change any state thats why. 
`   <FullCalendar
                header={false}
                columnHeaderFormat={{ weekday: "short" }}
                allDaySlot={false}
                height={337}
                defaultView="timeGridWeek"
                plugins={[timeGridPlugin]}
                events={Object.values(props.work_hours)}

            />`